nops-aws-account-register/README.md

```diff
@@ -14,6 +14,9 @@
 ### cloudformation-single-acc-register
 This folder contains CloudFormation template to register a single AWS account with nOps.
 
+### cloudformation-org-accounts-register
+This folder contains CloudFormation template to register AWS accounts with consolidated billing and organisation to nOps.
```

Comment:
Change to This folder contains **a** CloudFormation template..
Change organisation to: **organization**

I don't understand what you mean by "consolidated billing and organization to nOps". 
Can you explain it to me?

Reply:
done

Reply:
The sentence needs to be changed to:
This folder contains a CloudFormation template to register multiple AWS accounts within an organization, for consolidated billing.